Install/requirements.txt

```diff
@@ -1,3 +1,4 @@
+Cython==3.0.4
```

Comment:
Can remove this as well, this is defined in the IT8951 repo

Reply:
Agreed. That shouldn't be needed anymore since it's been fixed upstream.